draft-ietf-modpod-group-processes.md

```diff
@@ -328,8 +334,8 @@ consistent this memo and the guidelines approved by the IESG.  In cases
 of disagreement over a moderation decision, anyone may take the matter up
 with the responsible area director for resolution, or with the IETF chair
 if a responsible area director cannot be determined or is not assigned.
-Further appeals may be made to the IESG
-per {{Section 6.5.2 of !RFC2026}}, and then if necessary to the IAB.
+Further appeals may be made to the IESG per {{Section 6.5.2 of !RFC2026}},
+which also includes the option of an appeal to the IAB.
```

Comment:
```suggestion
Further appeals may be made to the IESG per {{Section 6.5.2 of !RFC2026}}.
```

As you know, the appeal to the IAB is not an option, it's an escalation. I'd actually prefer to not even mention that, since the pointer to 2026 should be sufficient.